src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -221,18 +226,31 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
         quiet.set(false)
         pull.set(false)
         cacheFrom.empty()
+
+        imageId.set(imageIdFile.map { RegularFile it ->
```

Comment:
We'll want to have similar handling in DockerCommitImage I'd think. Can you add that?

Reply:
I've added a fix for DockerCommitImage

Reply:
DockerCreateImage is also similar